src/pages/_proxied-dot-io/vault/home/index.tsx

```diff
@@ -0,0 +1,173 @@
+            ...card,
+            cta: card.cta[0],
+          }
+        })}
+      />
+
+      <IoHomeIntro
+        brand="vault"
+        heading={introHeading}
+        description={introDescription}
+        features={introFeatures}
+        video={{
+          youtubeId: _introVideo.youtubeId,
+          thumbnail: _introVideo.thumbnail.url,
+          heading: _introVideo.heading,
+          description: _introVideo.description,
+          person: {
+            name: _introVideo.personName,
+            description: _introVideo.personDescription,
+            avatar: _introVideo.personAvatar?.url,
+          },
+        }}
+      />
+
+      <section className={s.useCases}>
+        <div className={s.container}>
+          <IoCardContainer
+            heading={useCasesHeading}
+            description={useCasesDescription}
+            cardsPerRow={4}
+            cards={useCasesCards.map((card) => {
+              return {
+                eyebrow: card.eyebrow,
+                link: {
+                  url: card.link,
+                  type: 'inbound',
+                },
+                heading: card.heading,
+                description: card.description,
+                products: card.products,
+              }
+            })}
+          />
+        </div>
+      </section>
+
+      <IoHomeInPractice
+        brand="vault"
+        pattern="/vault/img/practice-pattern.svg"
+        heading={inPracticeHeading}
+        description={inPracticeDescription}
+        cards={inPracticeCards.map((card) => {
+          return {
+            eyebrow: card.eyebrow,
+            link: {
+              url: card.link,
+              type: 'inbound',
+            },
+            heading: card.heading,
+            description: card.description,
+            products: card.products,
+          }
+        })}
+        cta={{
+          heading: inPracticeCtaHeading,
+          description: inPracticeCtaDescription,
+          link: inPracticeCtaLink,
+          image: inPracticeCtaImage,
+        }}
+      />
+
+      <IoHomeCaseStudies
+        heading={caseStudiesHeading}
+        description={caseStudiesDescription}
+        primary={caseStudiesFeatured}
+        secondary={caseStudiesLinks}
+      />
+
+      <IoHomeCallToAction
+        brand="vault"
+        heading={callToActionHeading}
+        content={callToActionDescription}
+        links={callToActionCtas}
+      />
+
+      <IoHomePreFooter
+        brand="vault"
+        heading={preFooterHeading}
+        description={preFooterDescription}
+        ctas={preFooterCtas}
+      />
+    </>
+  )
+}
+Homepage.layout = VaultIoLayout
+
+export async function getStaticProps() {
+  const query = proxiedRivetClient('vault')
+  const { vaultHomepage } = await query({
+    query: homepageQuery,
+  })
+
+  return {
+    props: {
+      data: vaultHomepage,
+    },
+    revalidate:
+      process.env.HASHI_ENV === 'production'
+        ? process.env.GLOBAL_REVALIDATE
+        : 10,
+  }
```

Comment:
nit: we could put this value in the environment config

Reply:
This was carried over from the Vault repo, so I'm totally fine with just switching this to `__config('io_sites.revalidate')` instead of adding `GLOBAL_REVALIDATE`. Do we wanna drop the revalidate down to 10 seconds in development?

Reply:
Sure, I think for development revalidate won't come into play. For preview I think they want it intentionally low so authors can quickly see changes as they are made in Dato.

Reply:
👍 I'll update this PR to use the `__config`, and the follow up with another PR updating the values in dev and preview.